api-client/src/utils/generateUID.ts

```diff
@@ -0,0 +1,10 @@
+const DEFAULT_LENGTH = 20
+const generateUID = (length = DEFAULT_LENGTH): string => {
+  let generatedID = ''
+  const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789'
+  for (let i = 0; i < length; i += 1) {
+    generatedID += characters.charAt(Math.floor(Math.random() * characters.length))
```

Comment:
Is this random stuff really needed? I think a good way to handle the id would be: integreat-{event-id}-{last-updated}-{data}
Where we can drop the data-part, when introducing the rrule.
Generating a random-id would lead to problems, when you download an event you already downloaded and import both to a calander-app, if the event did not change, it should have the same id, so the calendar-app can detect it is the same event.

Reply:
Can't we safely hash the my suggested format? This would lead to same UUI if nothing changed

Reply:
added `// TODO IGAPP-1226: Replace by UID from CMS`

Reply:
alright i created a uuid using the url and lastUpdate